lib/matplotlib/axes/_base.py

```diff
@@ -2277,8 +2277,7 @@ def _process_unit_info(self, datasets=None, kwargs=None, *, convert=True):
 
     def in_axes(self, mouseevent):
         """
-        Return *True* if the given *mouseevent* (in display coords)
-        is in the Axes
+        Return whether the given event (in display coords) is in the axes.
```

Comment:
Why change Axes to axes? The former is the class, the latter is some non-specific thing.

Reply:
If you check the summary lines in https://matplotlib.org/devdocs/api/axes_api.html#module-matplotlib.axes, more often than not we use "axes" and not "Axes". Similar for "figure".

One could argue that there's a slight difference in meaning. "Axes/Figure" gives more weight to the type, and "axes/figure" is just addressing the general concept.

I'm not sure which one is best. The lower case version could be regarded as a bit sloppy. OTOH, the capitalized version seems a bit bulky. Personally, I feel the lower case version to read more smoothly, but of course that's subjective.

I'll revert this for now, to not block the PR. But it might be a good idea to have some rules on this in the docstring guide.

Reply:
Figure is okay, because there's only figure, but axes is ambiguous as it's a plural of axis. I've always tried to convert to Axes where possible if it seemed to be what was meant.

Reply:
Fair point. I've taken the discussion to #18726, because this itself PR does not change capitalization anymore.